cluster-autoscaler/metrics/metrics.go

```diff
@@ -270,7 +270,7 @@ var (
 			Namespace: caNamespace,
 			Name:      "scaled_down_gpu_nodes_total",
 			Help:      "Number of GPU nodes removed by CA, by reason and GPU name.",
-		}, []string{"reason", "gpu_name"},
+		}, []string{"reason", "resource_name", "gpu_name"},
```

Comment:
Do you know if it's safe to add a new label in the middle of the existing ones (e.g. I could imagine that some metric collector could treat earlier "gpu_name" values as "resource_name" after new 3-value metrics are emitted)?

Reply:
it's safe, it should be exposed in prometheus format with label names, so that metric collectors have no trouble identifying the right label